muted-tests.yml

```diff
@@ -82,8 +82,6 @@ tests:
 - class: org.elasticsearch.search.StressSearchServiceReaperIT
   method: testStressReaper
   issue: https://github.com/elastic/elasticsearch/issues/115816
-- class: org.elasticsearch.xpack.application.connector.ConnectorIndexServiceTests
```

Comment:
make sure to undo this change before merging (let's unmute this in a separate PR in case a revert will be needed later)